.github/actions/post-metric/action.yml

```diff
@@ -0,0 +1,51 @@
+name: Post Datadog Metric
+description: GitHub Action to post a metric to DD.
+
+inputs:
+  datadog-token:
+    description: 'token for datadog'
+    required: true
+  metric:
+    description: 'metric name for datadog'
+    required: true
+  metric-value:
+    description: 'metric value for datadog'
+    required: true
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Post Given Metric to Datadog using API.
+      shell: bash
+      id: post_metric
+      run: |
+        export NOW="$(date +%s)"
+        export DATADOG_API_KEY="${{ inputs.datadog-token }}"
```

Comment:
can these `export` commands be replaced with:
```
      env:
        DATADOG_API_KEY: ${{ inputs.datadog-token }}
        VALUE: ${{ inputs.metric-value }}
        METRIC: ${{ inputs.metric }}
```
? I don't know a lot about github's action syntax, but [this example](https://docs.github.com/en/actions/security-guides/encrypted-secrets#example-using-bash) suggests a pattern for transferring github context variables into shell environment variables, which provides slightly fewer opportunities to reveal the secret token. (I can see from the logs that github is doing a great job of censoring the tokens that it knows to be secret, but we might as well minimize the times it needs to do so).